runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -665,6 +673,55 @@ else if (message.contains(KILLED_TELEBLOCK_OPPONENT_TEXT))
 			}
 		}
 
+		if (config.showArceuus())
+		{
+			Duration duration = Duration.ofMillis(client.getRealSkillLevel(Skill.MAGIC) * Constants.GAME_TICK_LENGTH);
```

Comment:
Does boosting or draining your skill level affect the duration?

Reply:
As far as I can tell, boosts don't affect duration